schema/defs/vrs/Genotype.rst

```diff
@@ -31,4 +31,4 @@ Some Genotype attributes are inherited from :ref:`Variation`.
    *  - count
       - :ref:`Number` | :ref:`IndefiniteRange` | :ref:`DefiniteRange`
       - 1..1
-      - The total number of copies of all :ref:`MolecularVariation` at this locus, MUST be greater than or equal to the sum of :ref:`GenotypeMember` copy counts. If greater than the total counts, this implies additional  :ref:`MolecularVariation` that are expected to exist but are not explicitly indicated.
+      - The total number of copies of all :ref:`MolecularVariation` at this locus, MUST be greater than or equal to the sum of :ref:`GenotypeMember` copy counts and MUST be greater than or equal to 1. If greater than the total of GenotypeMember counts, this field describes  additional :ref:`MolecularVariation` that exist but are not  explicitly described.
```

Comment:
I am not sure I understand why the total nr of genotypemember counts can be different from the overall count. Do you have an example where this might be needed?

Reply:
The example we referenced when considering this use case was provided by @larrybabb, in [this example report from eMERGE](https://pubmed.ncbi.nlm.nih.gov/33930535/#&gid=article-figures&pid=fig-3-uid-2). What was found is that in these cases a heterozygous variant is reported, but no mention to the second allele (presumably reference) is given.